src/main/java/com/paneedah/mwc/proxies/ClientProxy.java

```diff
@@ -55,7 +55,7 @@ public void preInit(final MWC mod) {
     public void init(final MWC mod) {
         super.init(mod);
 
-        PlayerAnimations.init(mod);
+        MWC.modContext.setPlayerTransitionProvider(new PlayerTransitionProvider());
```

Comment:
Huh????

Reply:
Removed class PlayerAnimations, TransitionProvider needs to initialize somewhere still.